src/controllers/notifications.controller.js

```diff
@@ -31,6 +31,24 @@ const NotificationController = {
       'Notification status updated!'
     );
   },
+
+  async markAllNotificationsAsRead(req, res) {
+    const { userId } = req.userData;
+    const getAllNotifications = await NotificationService.getAllNotifications(userId);
```

Comment:
- We don't need to get all the notifications before applying the update.
- We can display a message when there are no notifications using the `GET` all `/notifications` route.